Dockerfile

```diff
@@ -1,15 +0,0 @@
-FROM eclipse-temurin:11-jre
-EXPOSE 8080
-
-ENV JAVA_TOOL_OPTIONS=""
-ENV JAVA_APP_JAR="${project.artifactId}.jar"
-
-VOLUME /tmp
-
-RUN mkdir /app && mkdir /opt/opentelemetry
-ADD target/docker-extra/opentelemetry-agent/opentelemetry-javaagent-${opentelemetry-agent.version}.jar /opt/opentelemetry/opentelemetry-javaagent.jar
```

Comment:
Will the new image build process also add the open telemetry java agent?

Reply:
the mvn spring-boot oci build doesn't add this javaagent, i'm not sure if the java-buildpack has an option for the OTEL-Agent or if we need to include a OTEL-Agent buildpack.

think it's ok to remove it for now.

keep in mind, that the spring-boot oci image requires 1 gb ram during execution (cgroup-limits), with less the application won't start! so this can be a breaking change